src/components/Rubric.js

```diff
@@ -51,7 +50,11 @@ const CompetencyButton = withRouter(props => {
               answer={criteria.answer}
               id={criteria.id}
               text={criteria.text} />
+          }).sort((a , b) => {
+            return a.id - b.id;
```

Comment:
as you are using this fn in several files, it might make sense to create a helper file and have the sort fn in there and pull it in in the various files where you use it.